src/components/interactive-builder/modals/question/question-form/rendering-types/inputs/select/select-answers.component.tsx

```diff
@@ -158,21 +215,17 @@ const SelectAnswers: React.FC = () => {
             label={t('searchForAnswerConcept', 'Search for a concept to add as an answer')}
             onSelectConcept={handleSelectAdditionalAnswer}
           />
-          {addedAnswers.length > 0 ? (
-            <div>
-              {addedAnswers.map((answer) => (
-                <Tag className={styles.tag} key={answer.id} type={'blue'}>
-                  {answer.text}
-                  <button
-                    className={styles.conceptAnswerButton}
-                    onClick={() => handleDeleteAdditionalAnswer(answer.id)}
-                  >
-                    X
-                  </button>
```

Comment:
By using a sortable tag, we've lost the feature to remove an added answer, which shouldn't be the case